src/main/kotlin/net/atos/zac/signalering/SignaleringService.kt

```diff
@@ -125,11 +125,28 @@ class SignaleringService @Inject constructor(
         signalering.targettype != SignaleringTarget.USER || signalering.target != actor
 
     @Transactional(REQUIRED)
-    fun createSignalering(signalering: Signalering): Signalering {
+    fun storeSignalering(signalering: Signalering): Signalering {
         ValidationUtil.valideerObject(signalering)
-        val created = entityManager.merge(signalering)
-        eventingService.send(ScreenEventType.SIGNALERINGEN.updated(created))
-        return created
+
+        val signaleringToStore = findSignalering(signalering)?.apply {
+            LOG.warning("Replacing $this timestamp $tijdstip with ${signalering.tijdstip}")
```

Comment:
I would make this loglevel INFO. It's normal behavior right? So not a warning I would say. We have INFO on on our test environment so we will see this come by.

Reply:
Done